src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java

```diff
@@ -5,8 +5,23 @@
 
 package com.microsoft.sqlserver.jdbc;
 
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.security.Key;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.UnrecoverableKeyException;
+import java.security.cert.CertificateEncodingException;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import java.text.MessageFormat;
+import java.util.Enumeration;
 import java.util.Locale;
 
+import javax.xml.bind.DatatypeConverter;
 
 /**
  * Provides the implementation of the key store provider for the Windows Certificate Store. This class enables using

```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck
**Severity**: ERROR
**File**: src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java L27

Line is longer than 80 characters (found 116).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<1777185663>*